assets/js/main.js

```diff
@@ -223,13 +223,13 @@ async function directory(jQuery) {
                     if(row.videochat && row.videochat.startsWith("http")) {
                         //or use custom url
                         return `
-                            <a href="${row.videochat}" targer="ohbm2020_${row.number}">
+                            <a href="${row.videochat}" targer="ohbm2020_${row.number}" target="room_${row.number}">
```

Comment:
so multiple targets is ok and handled consistently by the browsers (e.g taking the last one)? upon a quick google, didn't information

Reply:
Oops.. good catch. I completely missed that.

Reply:
Updated the PR